runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -42,16 +42,8 @@
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.ChatMessageType;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.Player;
-import net.runelite.api.Point;
-import net.runelite.api.SpriteID;
-import net.runelite.api.events.ChatMessage;
-import net.runelite.api.events.GameTick;
-import net.runelite.api.events.PlayerDeath;
-import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.*;
```

Comment:
Our code conventions are to avoid star imports. [Please read up on them and ensure your PR follows them.](https://github.com/runelite/runelite/wiki/Code-Conventions)

Reply:
adcb9e662a4dd734341b4a4cec2564dd8fdd4165